### PR TITLE
[API 5.1.0] Implement PVR_RECORDING.channelType support

### PR DESCRIPTION
**Pull request by ksooo**
@xhaggi mind taking a look

### CODE REVIEW

**Comment by ksooo** [comment]
@xhaggi ping

File: pvr.demo/PVRDemoAddonSettings.xml [hunk 1]
```diff
@@ -535,9 +536,141 @@
...
+      <time>14:00</time>
+      <duration>7500</duration>
+      <radio>0</radio>
+    </recording>
+    <!-- Radio Recordings -->
+    <recording>
+      <title>Demo Radio Recording entry 1</title>
+      <url></url>
+      <directory>/Directory1/SubDirectory1/</directory>
+      <channelname>Demo TV Channel 1</channelname>
```
Comment (xhaggi) [inline pvr.demo/PVRDemoAddonSettings.xml]: Demo _Radio_ Channel 1

File: pvr.demo/PVRDemoAddonSettings.xml [hunk 2]
```diff
@@ -535,9 +536,141 @@
...
+      <genresubtype>0</genresubtype>
+      <time>12:00</time>
+      <duration>7200</duration>
+      <radio>1</radio>
+    </recording>
+    <recording>
+      <title>Demo Radio Recording entry 2</title>
       <url></url>
       <directory>/Directory1/SubDirectory1/</directory>
       <channelname>Demo TV Channel 2</channelname>
```
Comment (xhaggi) [inline pvr.demo/PVRDemoAddonSettings.xml]: same here

File: pvr.demo/PVRDemoAddonSettings.xml [hunk 3]
```diff
@@ -547,9 +680,10 @@
...
       <time>14:00</time>
       <duration>7500</duration>
+      <radio>1</radio>
     </recording>
     <recording>
-      <title>Demo Recording entry 3</title>
+      <title>Demo Radio Recording entry 3</title>
       <url></url>
       <directory>/Directory1/SubDirectory1/</directory>
       <channelname>Demo TV Channel 3</channelname>
```
Comment (xhaggi) [inline pvr.demo/PVRDemoAddonSettings.xml]: same here and below

File: pvr.demo/PVRDemoAddonSettings.xml [hunk 4]
```diff
@@ -657,9 +800,141 @@
...
+      <time>14:00</time>
+      <duration>7500</duration>
+      <radio>0</radio>
+    </recording>
+    <!-- Deleted Radio Recordings -->
+    <recording>
+      <title>Demo Radio Recording entry 11</title>
+      <url></url>
+      <directory>/Directory1/SubDirectory1/</directory>
+      <channelname>Demo TV Channel 1</channelname>
```
Comment (xhaggi) [inline pvr.demo/PVRDemoAddonSettings.xml]: same here and below

File: pvr.demo/PVRDemoAddonSettings.xml (hunk 2 above)
Comment (ksooo) [inline pvr.demo/PVRDemoAddonSettings.xml]: Ooops. Thx.

File: pvr.demo/PVRDemoAddonSettings.xml (hunk 3 above)
Comment (ksooo) [inline pvr.demo/PVRDemoAddonSettings.xml]: And many more.... Got it. ;-)

**Comment by xhaggi** [comment]
after fixing the channel name for radio recordings you can merge it in.

**Comment by ksooo** [comment]
changed requested by @xhaggi are now in. merging now.

**Pull request merged**